src/middlewared/middlewared/plugins/chart_releases_linux/portals.py

```diff
@@ -63,7 +63,7 @@ def tag_func(key):
         cleaned_portals = {}
         for portal_type, schema in portals.items():
             t_portals = []
-            path = schema.get('path') or '/'
+            path = tag_func(schema.get('path')) or '/'
```

Comment:
It should be `tag_func(schema.get('path')  or '/')`

Reply:
> It should be `tag_func(schema.get('path') or '/')`

```suggestion
            path = tag_func(schema.get('path') or '/')
```